package.json

```
@@ -4,21 +4,24 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "build": "vite build",
+    "prebuild": "node .prebuild.js",
+    "build": "npm run prebuild && vite build",
```

```suggestion
    "build": "vite build",
```
npm offers `pre` and `post` scripts that will run automatically before or after a script.